platform/golden/bullseye.pkr.hcl

```diff
@@ -76,37 +90,36 @@ source "virtualbox-iso" "base-debian-amd64" {
   skip_nat_mapping     = false
   iso_checksum         = "${var.iso_checksum_type}:${var.iso_checksum}"
   iso_url              = "${var.iso_url}"
-  iso_interface        = "virtio"
+  iso_interface        = "sata"
   nic_type             = "virtio"
   hard_drive_interface = "virtio"
   rtc_time_base        = "UTC"
   shutdown_command     = "sudo -S shutdown -P now"
   ssh_username         = "packer"
   ssh_password         = "${var.ssh_password}"
   ssh_wait_timeout     = "30m"
+  firmware             = "efi"
+  # don't remove the built VM from VirtualBox after export
+  keep_registered      = "true"
+  # don't export the built VM
+  skip_export          = "false"
   vboxmanage = [
     # enable recording video of install process, for debug and build record
-    [ "modifyvm", "{{.Name}}", "--recording", "on" ],
```

Comment:
The recording is very valuable. Without it, we can't see what happens during early boot, specifically when `boot_command` is processed. Does the screen recording fail for you in some way?